src/applications/vaos/tests/appointment-list/components/RequestedAppointmentDetailsPage.unit.spec.jsx

```diff
@@ -0,0 +1,188 @@
+import {
+  mockFetch,
+  resetFetch,
+  setFetchJSONResponse,
+} from 'platform/testing/unit/helpers';
+
+import RequestedAppointmentDetailsPage from '../../../appointment-list/components/RequestedAppointmentDetailsPage';
+import { renderWithStoreAndRouter } from '../../mocks/setup';
+import {
+  getVAFacilityMock,
+  getVARequestMock,
+  getMessageMock,
+} from '../../mocks/v0';
+import { FETCH_STATUS } from '../../../utils/constants';
+import reducers from '../../../redux/reducer';
+import { transformPendingAppointments } from '../../../services/appointment/transformers';
+import { transformFacility } from '../../../services/location/transformers';
+
+const appointment = {
+  id: '1234',
+  ...getVARequestMock().attributes,
+  typeOfCareId: '323',
+  status: 'Submitted',
+  appointmentType: 'Primary care',
+  optionDate1: moment()
+    .add(3, 'days')
+    .format('MM/DD/YYYY'),
+  optionTime1: 'AM',
+  optionDate2: moment()
+    .add(4, 'days')
+    .format('MM/DD/YYYY'),
+  optionTime2: 'AM',
+  optionDate3: moment()
+    .add(5, 'days')
+    .format('MM/DD/YYYY'),
+  optionTime3: 'PM',
+  facility: {
+    ...getVARequestMock().attributes.facility,
+    facilityCode: '983GC',
+  },
+  bestTimetoCall: ['Morning'],
+  purposeOfVisit: 'New Issue',
+  email: 'patient.test@va.gov',
+  phoneNumber: '(703) 652-0000',
+  friendlyLocationName: 'Some facility name',
+};
+
+const pending = transformPendingAppointments([appointment]);
+
+const facility = {
+  ...getVAFacilityMock().attributes,
+  id: 'vha_442GC',
+  uniqueId: '442GC',
+  name: 'Cheyenne VA Medical Center',
+  address: {
+    physical: {
+      zip: '82001-5356',
+      city: 'Cheyenne',
+      state: 'WY',
+      address1: '2360 East Pershing Boulevard',
+    },
+  },
+  phone: {
+    main: '307-778-7550',
+  },
+};
+const var983GC = transformFacility(facility);
+
+const initialState = {
+  appointments: {
+    appointmentDetails: {},
+    appointmentDetailsStatus: FETCH_STATUS.notStarted,
+    requestMessages: {},
+    requestMessagesStatus: FETCH_STATUS.notStarted,
+    pendingStatus: FETCH_STATUS.succeeded,
+    pending,
+    facilityData: {
+      var983GC,
+    },
+  },
+};
+
+describe('VAOS <RequestedAppointmentDetailsPage>', () => {
+  beforeEach(() => {
+    mockFetch();
+    const message = getMessageMock();
+    message.attributes = {
+      ...message.attributes,
+      messageText: 'A message from the patient',
+    };
+
+    setFetchJSONResponse(
+      global.fetch.withArgs(
+        `${environment.API_URL}/vaos/v0/appointment_requests/1234/messages`,
+      ),
+      { data: [message] },
+    );
+  });
+  afterEach(() => resetFetch());
+
+  it('should render VA request details', async () => {
+    const screen = renderWithStoreAndRouter(
+      <Route path="/request/:id">
+        <RequestedAppointmentDetailsPage />
+      </Route>,
+      {
+        initialState,
+        reducers,
+        path: '/request/var1234',
```

Comment:
I think I'm going to put in a ticket to remove the `var` from the transformer for appointment/request ids. I don't think we want those in our urls and it's not really necessary to have for the app in general.